File: Ex3/Task3.9/FahrenheitToCelsius.scala
/**
 * Takes a temperature in degrees C
 * and prints out the same temperature in degrees F
 */
object FahrenheitToCelsius {
    def main(args: Array[String]) {
        var fahrenheit: Double = args(0).toDouble
        var celsius: Double = ((fahrenheit - 32) * 5) / 9

        println(fahrenheit + "F = " + celsius + "C")
    }
}

File: ex3/task3.9/FahrenheitToCelsius.java
/**
* Prints the converted value of Fahrenheit to Celsius 
* Results are always the expected ones
* @author Vlad Georgescu
*/
public class FahrenheitToCelsius
{
    public static void main(String[] args)
    {
        double Fahrenheit = Double.parseDouble(args[0]);
        double Celsius = (5 * Fahrenheit - 160) / 9;

        System.out.println("The converted value is " + Celsius);
    }
}